ui/src/app/app-routing.tsx

```diff
@@ -380,6 +382,14 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
               pageKey: 'conceptSetActions'
             }}/>}
           />
+          <AppRoute
+              path='/workspaces/:ns/:wsid/data/cohorts/build'
+              component={() => <CohortPagePage routeData={{
+                title: 'Build Cohort Criteria',
+                breadcrumb: BreadcrumbType.CohortAdd,
+                pageKey: 'cohortBuilder'
+              }}/>}
+          />
```

Comment:
In addition to using `withRouter`, I tried passing in `routeHistory` here like this:
```
<AppRoute
              path='/workspaces/:ns/:wsid/data/cohorts/build'
              component={({routeHistory}) => <CohortPagePage routeData={{
                title: 'Build Cohort Criteria',
                breadcrumb: BreadcrumbType.CohortAdd,
                pageKey: 'cohortBuilder'
              }}
              routeHistory={routeHistory}/>}
          />
```
 but again, never could get `routeHistory.block()` to fire.